pyproject.toml

```diff
@@ -1,2 +1,161 @@
+[project]
+name = "tinytuya"
+dynamic = ["version", "readme"]
+authors = [
+    { name = "Jason Cox", email = "jason@jasonacox.com" },
+]
+description = "Python module to interface with Tuya WiFi smart devices"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+
+dependencies = [
+    "colorama",
+    "requests",
+]
+
+requires-python = ">= 3.8"
+
+[project.optional-dependencies]
+cryptography = ["cryptography>=3.1"]
+pycryptodome = ["pycryptodome"]
+pyaes = ["pyaes"]
+pycrypto = ["pycrypto"]
+
```

Comment:
We had added the logic to setup.py for some small IoT devices/platforms that couldn't use some of these. I believe the intent was to allow the user to install one that did and the tinytuya iinstall would continue without insisting on cryptography. Yeah, I don't think it worked well. I do wonder how this would change that use case, if at all.